src/assets/i18n/dashboard/empty-portfolios-warning-modal/en.json

```diff
@@ -2,7 +2,7 @@
   "title": "Portfolios are missing",
   "explanation": "Some of the terminal's functionality is not available if you don't have portfolios with open positions or money.",
   "stepsTitle": "If you haven’t decided yet whether to transfer positions to us or not, but want to look at the terminal, you can do the following:",
-  "point1": "Through the client's account, credit 10 rubles via SBP to any of your portfolios.",
+  "point1": "Through the client's account, credit 500 rubles via SBP to any of your portfolios.",
```

Comment:
credit 500 rubles via SBP to any of your portfolios through the client's account